proposals/discard-parameters.md

```diff
@@ -0,0 +1,26 @@
+# Discard parameters
+
+## Summary
+
+Allow discards (`_`) to be used as parameters.
+For example:
+- lambdas: `(_, _) => 0`, `(int _, int _) => 0`
+- anonymous methods: `delegate(int _, int _) { return 0; }`
+- local functions: `void M() { void local(int _, int _) => ...; ... }`
+- methods: `override void M(int _, int _) { ... }`
+- indexers, constructors
+- delegate types: `delegate void MyDelegate(int _, int _);`
+
+## Motivation
+
+Unused parameters do not need to be named. The intent of discards is clear, i.e. they are unused/discarded.
+
+## Detailed design
+
+In a parameter list with more than one parameter named `_`, such parameters are discard parameters.
```

Comment:
> more than one parameter named `_` [](start = 24, length = 34)

The spec currently makes this an error. What spec changes are proposed to make it not be an error.

Reply:
"It is an error for two members of a method declaration space to have the same name."  ([method parameters](https://github.com/dotnet/csharplang/blob/master/spec/classes.md#method-parameters)).

I'll update speclet to say that discard parameters don't have a name (so the above rule doesn't apply).